composer.json

```diff
@@ -6,11 +6,13 @@
     "license": "MIT",
     "require": {
         "cakephp/console": "^4.0",
-        "rector/rector": "^0.14.5"
+        "rector/rector": "dev-main",
+        "webmozart/assert": "^1.11"
```

Comment:
this is needed as because the `webmozart/assert` that used in the rules no longer scoped